addons/slack/fusebit.json

```diff
@@ -1,3 +1,24 @@
 {
-  "fuseVersion": "1.8.4"
+  "security": {
+    "authentication": "optional",
+    "authorization": [
+      {
+        "action": "function:execute",
+        "resource": "/account/{{accountId}}/subscription/{{subscriptionId}}/boundary/{{boundaryId}}/functions/{{functionId}}/"
+      }
+    ],
+    "functionPermissions": {
+      "allow": [
+        {
+          "action": "storage:*",
+          "resource": "/account/{{accountId}}/subscription/{{subscriptionId}}/storage"
```

Comment:
What is this function doing that requires access to storage on the entire subscription? Ideally, the function instantiated from this fusebit.json would only have to operate on the part of the storage hierarchy it owns, e.g. `"/account/{{accountId}}/subscription/{{subscriptionId}}/storage/boundary/{{boundaryId}}/function/{{functionId}}/". 

At minimum, add a trailing slash.

Reply:
Same answer as above, this token needs to be broad enough so that when the addon instance asks for permissions to its storage, this token covers it.